dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -246,6 +248,59 @@ def test_add_profilers(self, *mocks):
         self.assertEqual("multiple files", merged_profile.encoding)
         self.assertEqual("multiple files", merged_profile.file_type)
 
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder." "ColumnPrimitiveTypeProfileCompiler"
+    )
+    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnStatsProfileCompiler")
+    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnDataLabelerCompiler")
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder.DataLabeler", spec=StructuredDataLabeler
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder."
+        "StructuredProfiler._update_correlation"
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder."
+        "StructuredProfiler._merge_correlation"
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder." "StructuredProfiler._update_chi2"
+    )
+    def test_add_profiles_null_count_not_enabled(self, *mocks):
+        profiler1_options_null_count = ProfilerOptions()
+        profiler1_options_null_count.set(
+            {"row_statistics.null_count.is_enabled": False}
+        )
+        profiler2_options_null_count = ProfilerOptions()
+        profiler2_options_null_count.set({"row_statistics.null_count.is_enabled": True})
+
+        data = pd.DataFrame([1, None, 3, 4, 5, None, 1])
+        with test_utils.mock_timeit():
+            profile1 = dp.StructuredProfiler(
+                data[:2], options=profiler1_options_null_count
+            )
+            profile2 = dp.StructuredProfiler(
+                data[2:], options=profiler2_options_null_count
+            )
+
+        self.assertEqual(0, profile1.row_has_null_count)
+        self.assertEqual(0, profile1.row_is_null_count)
+        self.assertLess(0, profile2.row_has_null_count)
+        self.assertLess(0, profile2.row_is_null_count)
+
+        try:
```

Comment:
Is something else erroring here and why we need the try? I would suggest not using it as then an exception could happen that we do not desire and it would silently fail.

The `with self.assertRaisesRegex` should capture the exception and not cause an error provided it matches the expected regex below.

Reply:
already on it and fixed it